src/electronicGroundState.c

```diff
@@ -62,6 +63,16 @@
 #define min(x,y) ((x)<(y)?(x):(y))
 #define TEMP_TOL 1e-12
 
+/**
+ * @brief   Calculate properties for fixed atom positions using DFT or MLFF
+ */
+void Calculate_Properties(SPARC_OBJ *pSPARC) {
```

Comment:
Tagging comment. Need to ad declaration in include/electronicGroundState.h

Reply:
added in 4c52587ee9bba69da4756ce7abfc6fdb038bd35d